crates/miden-objects/src/transaction/inputs/mod.rs

```diff
@@ -121,6 +121,11 @@ impl TransactionInputs {
     // CONVERSIONS
     // --------------------------------------------------------------------------------------------
 
+    /// Consumes these transaction inputs and returns underlying inputs.
+    pub fn into_input_notes(self) -> InputNotes<InputNote> {
+        self.input_notes
+    }
```

Comment:
Related to the last comment - I am not sure we need this method.

Reply:
Would prefer to keep this rather than performing the clone mentioned above.